### ISSUE TITLE
hasFeature("http://wwwindow.w3.org/TR/SVG11/feature#Image","1.1")

### ISSUE THREAD

**Issue by tobireif** [scottjehl/picturefill#654]
In the compacted lib there is
```
hasFeature("http://wwwindow.w3.org/TR/SVG11/feature#Image","1.1")
````

I think it should be
```
hasFeature("http://www.w3.org/TR/SVG11/feature#Image","1.1")
````

**Comment by jegtnes** [comment]
Hey @tobireif,

Thanks for the issue! You're certainly correct. This has been addressed in #609 and should be fixed if you upgrade to Picturefill 3.0.2. 👌

**Comment by tobireif** [comment]
Thanks @jegtnes !
